resotocore/resotocore/dependencies.py

```diff
@@ -184,6 +185,19 @@ def key_value(kv: str) -> Tuple[str, JsonElement]:
         "Note: this argument allows multiple overrides separated by space. "
         "Example: --override resotocore.api.web_hosts=localhost,some.domain resotocore.api.web_port=12345",
     )
+    parser.add_argument(
+        "--override-path",
+        nargs="+",
+        type=Path,
```

Comment:
It allows only files, not directories - idea was to have a directory with all my config overrides.